jbpm-test/src/main/resources/jBPM.properties

```diff
@@ -1,9 +1,9 @@
 # for creating a datasource
-persistence.datasource.name=jdbc/jbpm-ds
-persistence.datasource.user=sa
-persistence.datasource.password=
-persistence.datasource.url=jdbc:h2:tcp://localhost/~/jbpm-db
-persistence.datasource.driverClassName=org.h2.Driver
+name=jdbc/jbpm-ds
+driverClassName=org.h2.Driver
+user=sa
+password=
+url=jdbc:h2:tcp://localhost/~/jbpm-db
```

Comment:
Is JBPMHelper usable with these properties (although I know it is deprecated)? It still searches for properties with `persistence.datasource` prefix. Maybe we don't need to change this file at all.

Reply:
I didn't see that JBPMHelper was searching for properties with `persistence.datasource` prefix. I was about to submit a PR for simply these changes but I see this PR has made more changes. Hopefully the requestor will fix the formatting and this PR can be merged.

Reply:
@MarianMacik change from tabs to spaces in ChecklistExample.java otherwise it was imposible to put everything align.
done

Reply:
The problem is that eclipse (or devstudio) displays tab's as 4 spaces wide and github displays them as 8 spaces wide. I typically change DevStudio (or eclipse) to match that format so that it looks right on github or the rest of the world. Of course, if someone brings the code back into eclipse with the original settings it's messed up for them. The best solution I've found is set the tabs to 8 spaces but tell eclipse (or DevStudio) to always use spaces in the source.